guacamole-docker/bin/build-guacamole.sh

```diff
@@ -165,6 +165,21 @@ if [ -f extensions/guacamole-auth-duo/target/*.tar.gz ]; then
         "*.jar"
 fi
 
+#
+# Copy Saml auth extension if it was built
+#
+
+if [ -f extensions/guacamole-auth-saml/target/*.tar.gz ]; then
+    mkdir -p "$DESTINATION/saml"
+    tar -xzf extensions/guacamole-auth-saml/target/*.tar.gz \
+        -C "$DESTINATION/duo/"                             \
```

Comment:
> I've just been testing the changes and for some reason the .jar file for saml is not present in the docker image:

You are copying it to the wrong directory.  Copy paste error from the one above it ;)

Reply:
```suggestion
        -C "$DESTINATION/saml/"                             \
```